Jenkinsfile

```diff
@@ -22,6 +22,10 @@
 
 // mxnet libraries
 mx_lib = 'lib/libmxnet.so, lib/libmxnet.a, 3rdparty/dmlc-core/libdmlc.a, 3rdparty/tvm/nnvm/lib/libnnvm.a'
+
+// Python wheels
+mx_pip = 'mxnet_dist/*.whl'
```

Comment:
Are you sure about that path? in my experiments, it was in a different path: https://github.com/marcoabreu/incubator-mxnet/blob/add-raspberry-ci/tests/nightly/JenkinsfileEdge#L21